### PR TITLE
Fix UI Control focusing

### PR DESCRIPTION
**Pull request by ShadowCommander**
This fixes buttons not visually deselecting when dragging the mouse off of a button.

### CODE REVIEW

File: Robust.Client/UserInterface/UserInterfaceManager.cs [hunk 1]
```diff
@@ -168,6 +168,7 @@ public void InitializeTesting()
 
         public void Dispose()
         {
+            _controlFocused?.Dispose();
```
Comment (PJB3005) [inline Robust.Client/UserInterface/UserInterfaceManager.cs]: Why is this necessary?

Comment (ShadowCommander) [reply Robust.Client/UserInterface/UserInterfaceManager.cs]: It's a Visual Studio suggestion, I can remove it.
